sections/shared/Layout/SideNav/DesktopSideNav.tsx

```diff
@@ -76,10 +76,12 @@ const DesktopSideNav: FC = () => {
 					<CRatioBarStats />
 					<BalanceItem amount={snxBalance} currencyKey={CryptoCurrency.SNX} />
 					<BalanceItem amount={sUSDBalance} currencyKey={Synths.sUSD} />
-					<StyledTargetStakingRatio>
-						<StyledTargetStakingRatioTitle>Staking %</StyledTargetStakingRatioTitle>
-						{tRatio ? tRatio : '0.00'}%
-					</StyledTargetStakingRatio>
+					<Tooltip content={t('common.total-staking.staking-percentage')}>
+						<StyledTargetStakingRatio>
+							<StyledTargetStakingRatioTitle>Staking %</StyledTargetStakingRatioTitle>
+							{tRatio ? tRatio : '0.00'}%
```

Comment:
```tRatio || '0.00'```

Reply:
haha that's a nit pick if I've ever seen one.. Made the change and added translation for `Staking %`